common/achievements/no-derpcoins.ts

```diff
@@ -17,12 +15,8 @@ const NoDerpcoins: Achievement = {
 		},
 	],
 	onGameStart(game, player, component, observer) {
-		if (!game.state.isEvilXBossGame) return
-		const cost = getDeckCost(
-			game.components
-				.filter(CardComponent, query.card.player(player.entity))
-				.map((card) => card.props),
-		)
+		const cost = getDeckCost(player.getDeck().map((card) => card.props))
```

Comment:
I think @Screaper91 used that component query for a reason

Reply:
This should be the same thing. I think `getDeck()` is easier to look at.